Document/0x05b-Basic-Security_Testing.md

```diff
@@ -58,7 +58,14 @@ See also "Verifying the Variability of Tampering Responses" for further details.
 
 ##### Rooting your device
 
--- TODO : Maybe add a warning on rooting devices (brickification, additional security risks, warranty nullified, ...), disclaimer --
+###### Risks of rooting your device
+
+As a security tester, you may want to root your mobile device, all the more so as it can make performing some tests easier, when using a rooted device is not mandatory. However, you need to be aware of the fact that rooting is not an easy process and requires advanced knowledge. Rooting is risky, and three main consequences need to be clarified before you may proceed: rooting
```

Comment:
Maybe make close the sentence before "when using a rooted...", for example:

As a security tester, you may want to root your mobile device, all the more so as it can make performing some tests easier or even allows testing as bypassing of implemented security controls is possible. Using a rooted device is not mandatory, however...